### ISSUE TITLE
WCAG issue - Click targets of < > buttons for browsing filters are confusing and too small

### ISSUE THREAD

**Issue by Rainribbon** [Kaltura-EDU/issues-tracking#11]
My Media and Media Gallery include what appears to be a single button, "Browse Filters." However, the functional buttons are very small "Next" ( > ) and "Back" ( < ) navigation arrow buttons. This is confusing because it looks to a user like everything included within the outlined "button" region around the "Browse Filters" text is a clickable button, when it is not. Additionally, the clickable area for each navigation arrow button seems to fail [WCAG SC 2.5.8 - Target size (minimum)](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html) because the buttons appear to be 18 x 18 pixels, and the minimum target size is 24 x 24.

![Image](https://github.com/user-attachments/assets/47349091-0945-4c85-aceb-02d2dc30ccc4)

Steps to reproduce:
1. In any KAF, navigate to My Media or a Media Gallery that includes some media entries.
2. Select "Filters".
3. Try to click in the middle of the "Browse Filters" button and it will appear to be broken because the text is not clickable.
4. Make your browser window narrower (about half your normal screen size) or zoom in a bit in the browser. **Note:** Narrower browser width or being zoomed in is required to trigger the appearance of the Browse Filters "button" that includes arrow buttons pointing in both directions.
5. Note that you need to select the > button to navigate to the next filters, and the click target for this > is less than the minimum 24 x 24 pixels required for WCAG SC 2.5.8.

Kaltura has filed the following ticket for this issue: **ADA-2789 | “Browse Filters” navigation arrows are misleading and fail WCAG target size requirement** with the following recommendations:

- Make the entire "Browse Filters" container clickable to reduce confusion.
- Ensure arrow buttons meet the minimum target size (24 x 24 px).
- Alternatively, consider replacing the two-arrow UI with clearly labeled buttons like “Previous Filters” and “Next Filters” to improve usability and clarity.
